### DISCUSSION TITLE
'default' log file

### DISCUSSION THREAD

**Discussion by staabm** [deployphp/deployer#2382] [Help needed]
the `dep` command supports a `--log` option which allows to pass in a filename to log into.

https://github.com/deployphp/deployer/blob/367c9bf7230ec22f60ba6223bcb0c9b7879ccf58/src/Command/MainCommand.php#L73-L78

is there a way to define this log file from e.g. within the recipe or similar, so I don't always need to pass it via cli switches?

**Comment by antonmedv** [comment]
I believe it's possible with `set('log_file', '...');`

**Reply by staabm** [reply]
perfect, this works.

should this be documented somewhere?

**Reply by antonmedv** [reply]
Yes, I'm working on new documentation right now. +1
